rfcs/README.md

```diff
@@ -0,0 +1,116 @@
+# RFCSs
+
+The RFC process here is inspired by [that of the Rust programming
+language](https://github.com/rust-lang/rfcs).
+
+The process is intended to provide TRC team members a venue in which to propose substantial
```

Comment:
```suggestion
The process is intended to provide CTD team members a venue in which to propose substantial
```

Reply:
Just noticed the "Commit suggestion" feature here that I could have used. Nifty!